src/main/java/kr/pickple/back/alarm/dto/request/CrewAlarmUpdateStatusRequest.java

```diff
@@ -2,13 +2,19 @@
 
 import jakarta.validation.constraints.NotNull;
 import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
`p3`

- 롬복을 사용하고 있으니 아래와 같이 조금 더 간결하게 표현할 수 있을 것 같아요! 👍 👍

```suggestion
@AllArgsConstructor(access = AccessLevel.PRIVATE, staticName = "from")
```

Reply:
그렇다면 이전 리뷰를 남기신 `access = AccessLevel.PRVIATE`는 삭제 해야 됩니다! (테스트 환경에서 사용해야 하니깐요!)
해당 부분 `access`삭제하고 `staticName = from`으로 수정하였습니다~
반영 커밋: 79bea8d63f1d86ec7e42c3b0ebb8ed399ace40b6